### ISSUE TITLE
Spikes in training curve

### ISSUE THREAD

**Issue by andreydung** [vitoralbiero/img2pose#46]
Hi Vitor,

I'm using the default code and the default WiderFace data. However training curve is having spikes as in the following image.

![Screenshot from 2021-09-04 11-40-10](https://user-images.githubusercontent.com/1322198/132082654-54497638-56d2-4ff5-88df-198a273d5bab.png)

I'm wondering if this is abnormal or not?

**Comment by vitoralbiero** [comment]
Hello @andreydung,

That's normal, I also had a lot of spikes during training. If the loss is trending down, everything is good. This is mostly caused by the small batch size used.
Check the validation loss, which shouldn't have spikes and also trend down.

Hope this helps.
